pycontw/pycon.tw
# chore(proposal): update the links of speaking-related pages

## Types of changes

- [x] **Other (update outdated links)**

## Description

The speaking-related pages are now migrated to an individual project (see the repo [here](https://github.com/pycontw/pycontw-2021)). The external links to these pages in the proposal system should be updated to the new ones.

## Steps to Test This Pull Request
Steps to reproduce the behavior:
1. Go to `<host>/proposals/talk/submit/` & `<host>/proposals/tutorial/submit/`
2. Click on the external links in the help text and alert section.
3. It should link to the new pages.

## More Information
The pictures below show where the updated links are:

alert section:
![image](https://user-images.githubusercontent.com/24987826/109331790-3f761f00-7898-11eb-8bac-1f0ee221c879.png)

help text:
![image](https://user-images.githubusercontent.com/24987826/109331682-1b1a4280-7898-11eb-8097-2366589bc51b.png)
